src/react/components/InputGroup.jsx

```diff
@@ -1,4 +1,3 @@
-import { Input } from 'antd';
-import 'antd/es/input/style/index.js';
+import { Space } from 'antd';
 
-export const InputGroup = Input.Group;
+export const InputGroup = Space.Compact;
```

Comment:
Is this copy/paste or valid?

Reply:
It's valid as Input.Group was deprecated and there was a warning to replace it with Space.Compact.